WcaOnRails/app/models/country.rb

```diff
@@ -1,1870 +1,7 @@
-      "zoom": 0,
-      "iso2": "UA",
-    },
-    {
-      "id": "United Arab Emirates",
-      "name": "United Arab Emirates",
-      "continentId": "_Asia",
-      "latitude": 0,
-      "longitude": 0,
-      "zoom": 0,
-      "iso2": "AE",
-    },
-    {
-      "id": "United Kingdom",
-      "name": "United Kingdom",
-      "continentId": "_Europe",
-      "latitude": 55378051,
-      "longitude": -3435973,
-      "zoom": 5,
-      "iso2": "GB",
-    },
-    {
-      "id": "Uruguay",
-      "name": "Uruguay",
-      "continentId": "_South America",
-      "latitude": 0,
-      "longitude": 0,
-      "zoom": 0,
-      "iso2": "UY",
-    },
-    {
-      "id": "USA",
-      "name": "USA",
-      "continentId": "_North America",
-      "latitude": 37090240,
-      "longitude": -95712891,
-      "zoom": 4,
-      "iso2": "US",
-    },
-    {
-      "id": "Uzbekistan",
-      "name": "Uzbekistan",
-      "continentId": "_Asia",
-      "latitude": 0,
-      "longitude": 0,
-      "zoom": 0,
-      "iso2": "UZ",
-    },
-    {
-      "id": "Vanuatu",
-      "name": "Vanuatu",
-      "continentId": "_Oceania",
-      "latitude": 0,
-      "longitude": 0,
-      "zoom": 0,
-      "iso2": "VU",
-    },
-    {
-      "id": "Venezuela",
-      "name": "Venezuela",
-      "continentId": "_South America",
-      "latitude": 0,
-      "longitude": 0,
-      "zoom": 0,
-      "iso2": "VE",
-    },
-    {
-      "id": "Vietnam",
-      "name": "Vietnam",
-      "continentId": "_Asia",
-      "latitude": 0,
-      "longitude": 0,
-      "zoom": 0,
-      "iso2": "VN",
-    },
-    {
-      "id": "XA",
-      "name": "Multiple Countries (Asia)",
-      "continentId": "_Asia",
-      "latitude": 0,
-      "longitude": 0,
-      "zoom": 0,
-      "iso2": "XA",
-    },
-    {
-      "id": "XE",
-      "name": "Multiple Countries (Europe)",
-      "continentId": "_Europe",
-      "latitude": 0,
-      "longitude": 0,
-      "zoom": 0,
-      "iso2": "XE",
-    },
-    {
-      "id": "XS",
-      "name": "Multiple Countries (South America)",
-      "continentId": "_South America",
-      "latitude": 0,
-      "longitude": 0,
-      "zoom": 0,
-      "iso2": "XS",
-    },
-    {
-      "id": "Yemen",
-      "name": "Yemen",
-      "continentId": "_Asia",
-      "latitude": 0,
-      "longitude": 0,
-      "zoom": 0,
-      "iso2": "YE",
-    },
-    {
-      "id": "Zambia",
-      "name": "Zambia",
-      "continentId": "_Africa",
-      "latitude": 0,
-      "longitude": 0,
-      "zoom": 0,
-      "iso2": "ZM",
-    },
-    {
-      "id": "Zimbabwe",
-      "name": "Zimbabwe",
-      "continentId": "_Africa",
-      "latitude": 0,
-      "longitude": 0,
-      "zoom": 0,
-      "iso2": "ZW",
-    },
-  ].map { |e| Country.new(e) }
-
-  ALL_COUNTRIES_BY_ID = Hash[ALL_COUNTRIES.map { |e| [e.id, e] }]
-  ALL_COUNTRIES_BY_ISO2 = Hash[ALL_COUNTRIES.map { |e| [e.iso2, e] }]
-  ALL_COUNTRIES_WITH_NAME_AND_ID = Hash[ALL_COUNTRIES.map { |e| [e.name, e.id] }]
```

Comment:
If this data won't change without a reboot, we can keep these constants:

`ALL_COUNTRIES_WITH_NAME_AND_ID = Country.all.map { |country| [country.name, country.id] }.freeze`

...and so on for the other constants, assuming they still make sense.